operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/RetryInfo.java

```diff
@@ -1,8 +1,22 @@
 package io.javaoperatorsdk.operator.api;
 
+/**
+ * Interface to be implemented by user-defined classes encapsulating information about how many
```

Comment:
This isn't true: this isn't a class that users need to implement. It's simply providing a contract so that we don't expose internal implementation classes. I'm not even sure this class needs to be exposed to users at all, to be honest.